portable/GCC/ARM_CM4F/port.c

```diff
@@ -101,6 +94,19 @@ occurred while the SysTick counter is stopped during tickless idle
 calculations. */
 #define portMISSED_COUNTS_FACTOR			( 45UL )
 
+/* Let the user override the default SysTick clock rate.  If defined by the
+user, this symbol must equal the SysTick clock rate when the CLK bit is 0 in the
+configuration register. */
+#ifndef configSYSTICK_CLOCK_HZ
+	#define configSYSTICK_CLOCK_HZ configCPU_CLOCK_HZ
+	/* Ensure the SysTick is clocked at the same frequency as the core. */
+	#define portNVIC_SYSTICK_CLK_BIT_SETTING	( portNVIC_SYSTICK_CLK_BIT )
+#else
+	/* Select the option to clock SysTick not at the same frequency as the core.
+	The clock used is often a divided version of the core clock. */
+	#define portNVIC_SYSTICK_CLK_BIT_SETTING	( 0 )
+#endif
+

```

Comment:
I may have caused a readability issue here with the `_SETTING` suffix.  It's not an established naming convention in FreeRTOS code.  The intention of the `_SETTING` suffix is to allow `portNVIC_SYSTICK_CLK_BIT` to have a fixed, expected value, regardless of build configuration.  The new code at line 694 uses this bit regardless of build configuration.  Would a `_CONFIG` suffix be more readable?  Is there some other convention already established in FreeRTOS code that we could use instead?